src/Microsoft.AspNetCore.HttpsPolicy/HstsOptions.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.HttpsPolicy
+{
+    /// <summary>
+    /// Options for the Hsts Middleware
+    /// </summary>
+    public class HstsOptions
+    {
+        /// <summary>
+        /// Sets the max-age parameter of the Strict-Transport-Security header.
+        /// </summary>
+        /// <remarks>
+        /// Max-age is required; defaults to 0.
+        /// See: https://tools.ietf.org/html/rfc6797#section-6.1.1
+        /// </remarks>
+        public int MaxAge { get; set; } // TODO set this to a different default in dev/not dev?
```

Comment:
0 cannot be the default, that has a specific meaning in the spec. Ask the PMs for a specific value. 1 day? 1 week?
We may not need a separate value for dev as the templates won't have this enabled by default.

Reply:
Discussed offline with @javiercn. 0 isn't a valid default, and using a nullable would unnecessarily complicate the user-experience. We just need to come up with a good default. Take it up with @blowdart over email if needs be.

Reply:
@shirhatti as I know you were opinionated on having 0 as the default.